e2e-tests/cypress/support/check-value.js

```diff
@@ -0,0 +1,3 @@
+export default (selector, expectedValue) => {
+  selector.should('have.value', expectedValue);
+};
```

Comment:
Great stuff - what do you think about having the selector being executed inside the command? So instead of having this (current):

```js
cy.checkValue(broker[NAME].input(), 'mock value');
```

It could be 

```js
cy.checkValue(broker[NAME].input, 'mock value');
```


so no `input()` in tests - think it reads nicer in the tests and easier to type?